articles/search/search-filters.md

```diff
@@ -67,23 +67,23 @@ The following examples represent prototypical filter definitions in several APIs
 
 ```http
 # Option 1:  Use $filter for GET
-GET https://[service name].search.windows.net/indexes/hotels/docs?search=*&$filter=baseRate lt 150&$select=hotelId,description&api-version=2019-05-06
+GET https://[service name].search.windows.net/indexes/hotels/docs?search=*&$filter=Rfilter=Rooms/any(room: room/BaseRate lt 150.0)&$select=HotelId,Description&api-version=2019-05-06
 
 # Option 2: Use filter for POST and pass it in the request body
 POST https://[service name].search.windows.net/indexes/hotels/docs/search?api-version=2019-05-06
 {
     "search": "*",
-    "filter": "baseRate lt 150",
-    "select": "hotelId,description"
+    "filter": "Rooms/any(room: room/BaseRate lt 150.0",
+    "select": "HotelId,Description"
```

Comment:
please extend the select to include Rooms/Description and Rooms/BaseRate as a verification measure